odoo/fields.py

```diff
@@ -1747,22 +1792,23 @@ def get_text_content(self, term):
         func = getattr(self.translate, 'get_text_content', lambda term: term)
         return func(term)
 
-    def convert_to_column(self, value, record, values=None, validate=True):
+    def convert_to_column(self, value, record, values=None, validate=True, operation='create'):
+        if operation == 'flush':
+            """ Convert from cache_raw value to column value """
+            return PsycopgJson(value) if (self.translate or self.company_dependent) and value else value
+        if operation == 'search':
+            return self.convert_to_cache(value, record)
         cache_value = self.convert_to_cache(value, record, validate)
+        if self.company_dependent:
```

Comment:
There should be an error during the initialization if you have a translated, company_dependent field (I haven't found one in the code).

Reply:
the check is in `def _get_attrs` of `Fields`, but I disable it for now